web-components/src/components/menu-overlay/scss/settings.scss

```diff
@@ -1,5 +1,5 @@
 $arrow-size: rem-calc(16);
-$arrow-offset: rem-calc(2) - $arrow-size;
+$arrow-offset: rem-calc(0.8) - $arrow-size;
```

Comment:
Arrow was sitting behind the menu prior to this change, which was essentially causing the border issue, so this value did not need to be very accurate. Adjusting this to accurately align with edge of menu.